tasks/src/it/scala/com/itv/bucky/tasks/TaskFs2IntegrationTest.scala

```diff
@@ -0,0 +1,74 @@
+package com.itv.bucky.tasks
+import _root_.fs2.Stream
+import cats.effect.IO
+import cats.implicits._
+import com.itv.bucky._
+import com.itv.bucky.fs2._
+import org.scalatest.FunSuite
+import cats.effect.IO.contextShift
+import com.itv.bucky.UnmarshalResult.Success
+
+class TaskFs2IntegrationTest
```

Comment:
does this feature need an integration test? as it's just constructing instances of publishers / consumers it could be tested at the unit level using the simulator (this would also help get rid of the FS2 dependency)

Reply:
I never looked at the simulator. In AM world we don't use the simulator and usually unit test handlers directly and integration test the wiring against Qpid or Rabbit. This is mainly about testing the wiring.

Reply:
Think that can be done in a way similar to this without the FS2 dependency:
https://github.com/ITV/bucky/blob/master/rabbitmq/src/test/scala/com/itv/bucky/future/GenericPublisherTest.scala